cdc/redo/manager.go

```diff
@@ -375,8 +375,20 @@ func (m *ManagerImpl) AddTable(tableID model.TableID, startTs uint64) {
 
 // RemoveTable removes a table from redo log manager
 func (m *ManagerImpl) RemoveTable(tableID model.TableID) {
```

Comment:
Please add some comments here to explain why flush ts in the rtsmap is necessary